pkg/grafana/dashboards_test.go

```diff
@@ -11,22 +11,12 @@ import (
 )
 
 func TestDashboard(t *testing.T) {
```

Comment:
Do we have any tests that run the actual grr commands? I feel like these tests are slightly short of testing the whole thing and it'd actually be more useful (full integration test) and easier to call the `grr` commands to do the work